src/components/features/about-section/about-section.tsx

```
@@ -0,0 +1,18 @@
+import AboutImages from "./components/about-images";
+import AboutContent from "./components/about-content";
+
+export default function AboutSection() {
+    return (
+        <section className="flex min-h-[399px] gap-20">
```

ESLint

[error] className value must use cn() function. Example: className={cn("...")}

(local-rules/require-cn-classname)